### PR TITLE
Remove new moderation flow feature flag

### PR DESCRIPTION
**Pull request by pelumy**
## Issues covered
#1646

## Description
Removes the "new moderation flow" feature flag and old moderation flow code.

## How to test
1. Build the code.
2. Navigate to settings screen.
3. Please check that there is no "new moderation flow toggle", and the new moderation flow is the one being displayed.

## Screenshots/Video
| Before | After |
|--------|-------|
| Before | After |

### CODE REVIEW

**Comment by joshuatbrown** [comment]
👀

**Review by joshuatbrown — COMMENTED** [review]
I think we can remove a lot of localized strings, too -- maybe all of Moderation.xcstrings. Could you check that and see what we can remove, or if we can remove all of it?

File: Nos/Views/Modifiers/ReportMenuModifier.swift
[the remark this answers is not in the document]
Comment (pelumy) [reply Nos/Views/Modifiers/ReportMenuModifier.swift:1]: 🤣

**Comment by pelumy** [comment]
> I think we can remove a lot of localized strings, too -- maybe all of Moderation.xcstrings. Could you check that and see what we can remove, or if we can remove all of it?

You are soo right! I will do that.

**Comment by pelumy** [comment]
@joshuatbrown , I checked the code and I don't think we can delete the localised strings yet because the `WarningController` is using it via the `ReportCategory`. We will need to refactor `ReportCategory` to be able to delete them.

**Review by mplorentz — APPROVED** [review]
This looks good to me! We can delete the `enableNewModerationFlow` key from Localizable.strings, but after that I'm good to merge!

**Comment by pelumy** [comment]
> enableNewModerationFlow

🧙‍♂️ This is true! Thanks for catching this @mplorentz

**Pull request merged**